crates/oxc_macros/src/declare_oxc_lint.rs

```diff
@@ -67,3 +70,21 @@ pub fn declare_oxc_lint(metadata: LintRuleMeta) -> TokenStream {
 
     output
 }
+
+/// Convert a struct identifier's string (pascal case) to kebab case
+/// Done so names will match the canonical ES/TSLint rule names
+fn convert_to_kebab_case(original_string: &str) -> String {

```

Comment:
We can use https://crates.io/crates/convert_case for simplicity, we'll probably need the other case conversions later.